packages/terra-profile-image/docs/README.md

```diff
@@ -0,0 +1,21 @@
+# Terra Image
+
+The terra-profile-image component displays an avatar image while the profile image is being loaded. If the profile image successfully loads, the avatar image is replaced with the profile image. Otherwise, the avatar image is left as is. All styling set on the profile image will be applied to the avatar image.
+
+## Getting Started
+
+- Install with [npmjs](https://www.npmjs.com):
+  - `npm install terra-image`
+  - `yarn add terra-image`
+
+## Usage
+
+```jsx
+import React from 'react';
+import ProfileImage from 'terra-profile-image';
+import Image from 'terra-image';
```

Comment:
Yeah, we could definitely update this component to replicate the props from image (I assume we would want more than just src so styling could be applied).  I brought this question up here [1] and the consensus was to just pass in an image.

[1] https://github.com/cerner/terra-core/issues/429#issuecomment-308837639